tutorial/source/dirichlet_process_mixture.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
You would need to add this tutorial to the Contributed section of the file https://github.com/pyro-ppl/pyro/blob/dev/tutorial/source/index.rst